### PR TITLE
add the kadena docs app

### PR DESCRIPTION
**Pull request by sstraatemans**
adding the kadena docs app to the mono repo

- [x] I have reviewed my changes and run the appropriate tests.
- [x] I have have run `rush change` to add the appropriate change logs.
- [ ] I have added/edited docs.
- [ ] I have added tutorials.
- [ ] I have double checked and DEFINITELY added docs.

### CODE REVIEW

File: packages/apps/kadena-docs/.storybook/preview.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import {
+  darkTheme,
+  globalCss,
+  baseGlobalStyles,
+} from '@kadena-ui/react-components';
+import { themes } from '@storybook/theming';
+
+console.log(darkTheme);
```
Comment (alber70g) [inline packages/apps/kadena-docs/.storybook/preview.js:8]: Remove?

File: packages/apps/kadena-docs/config/rig.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/rig-package/rig.schema.json",
+
+  "rigPackageName": "@kadena-dev/heft-rig"
```
Comment (alber70g) [inline packages/apps/kadena-docs/config/rig.json:4]: Maybe we can use @rushstack/heft-web-rig/profiles/web
See also https://app.asana.com/0/1202676524474548/1204337512939263

File: packages/apps/kadena-docs/package.json [hunk 3]
```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@kadena/kadena-docs",
+  "version": "0.1.0",
+  "scripts": {
+    "_phase:build": "next build",
+    "_phase:test": "",
```
Comment (alber70g) [inline packages/apps/kadena-docs/package.json:6]: Maybe use `npm run build` and `npm run test` that way we have the same build local as in CI

File: packages/apps/kadena-docs/package.json [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    "build": "next build",
+    "cypress:open": "cypress open",
+    "dev": "next dev",
+    "lint": "next lint",
+    "start": "next start",
+    "storybook": "storybook dev -p 6006",
+    "test": "jest"
+  },
+  "dependencies": {
```
Comment (alber70g) [inline packages/apps/kadena-docs/package.json:15]: Can you add the lint stuff, see: https://github.com/kadena-community/kadena.js/blob/master/packages/apps/explorer/package.json#L11-L24

File: packages/apps/kadena-docs/package.json [hunk 5]
```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@kadena/kadena-docs",
+  "version": "0.1.0",
+  "scripts": {
+    "_phase:build": "next build",
+    "_phase:test": "",
+    "build": "next build",
+    "cypress:open": "cypress open",
+    "dev": "next dev",
+    "lint": "next lint",
```
Comment (alber70g) [inline packages/apps/kadena-docs/package.json:10]: Maybe we should use `"lint": "npx eslint ./src --ext .js,.ts --fix",`
https://github.com/kadena-community/kadena.js/blob/master/packages/libs/client/package.json#L17
If this doesn't work we can fix it when implementing the heft-rig for next apps

File: packages/apps/kadena-docs/tsconfig.json [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+    "skipLibCheck": true,
+    "allowJs": true,
+    "noEmit": true,
+    "incremental": true,
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "jsx": "preserve"
+  },
+  "exclude": ["node_modules"]
```
Comment (alber70g) [inline packages/apps/kadena-docs/tsconfig.json:14]: I think this is excluded in the `tsconfig-base`. Maybe check all props and remove duplicates.
It's preferred to have nothing here, so check if the build still works with everything removed except types, I assume

File: rush.json [hunk 7]
```diff
@@ -427,6 +427,14 @@
       "shouldPublish": false
     },
 
+    {
+      "packageName": "@kadena/kadena-docs",
+      "projectFolder": "packages/apps/kadena-docs",
+      "reviewCategory": "prototypes",
+      "skipRushCheck": true,
```
Comment (alber70g) [inline rush.json:434]: Why do we `skipRushCheck`?
* https://rushjs.io/pages/configs/rush_json/#:~:text=//%20%22skipRushCheck%22%3A%20false
* https://rushjs.io/pages/commands/rush_check/

It shouldn't be necessary. Maybe for apps it's ok to use though

**Review by alber70g — DISMISSED** [review]

**Review by jermaine150 — APPROVED** [review]

**Pull request merged**